src/main/java/svc/models/VIOLATION_STATUS.java

```diff
@@ -0,0 +1,23 @@
+package svc.models;
+
+public enum VIOLATION_STATUS{
+	CONT_FOR_PAYMENT ("Continued for payment"),
+	FTA_WARRANT_ISSUED ("An arrest warrent has been issued for failure to appear to your court date"),
+	DISMISS_WITHOUT_COSTS ("Dismissed"),
+	CLOSED ("Closed");
+	
+	private final String msg;
+	
+	VIOLATION_STATUS (String msg){
+		this.msg = msg;
+	}
+	
+	@Override
+	public String toString(){
```

Comment:
Does switching to VIOLATION_STATUS and providing this '.toString()' here affect the API return values for the violation controller? I bet it serializes these statuses to their messages instead of the old DB representations that the UI is probably expecting.